Package.swift

```diff
@@ -106,6 +107,8 @@ let package = Package(
         // Use a forked repository of the https://github.com/apple/swift-metrics-extras repository that
         // is versioned and already contains test functionality
         .package(url: "https://github.com/Apodini/swift-metrics-extras.git", .upToNextMinor(from: "0.1.0")),
+        .package(url: "https://github.com/apple/swift-distributed-tracing.git", .upToNextMinor(from: "0.1.2")),
+        .package(url: "https://github.com/slashmo/opentelemetry-swift.git", from: "0.1.1"),
```

Comment:
Probably would also make sense to use `.upToNextMinor(from:` here.

Reply:
Agreed, thanks for pointing it out!